portfolio/src/main/webapp/map.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  console.log("map createing")
+  const map = new google.maps.Map(
```

Comment:
Do you need to return this or store it or something?

Reply:
Right now, no. ln the future, maybe?